extensions/wikia/Email/Email.setup.php

```diff
@@ -52,11 +52,11 @@
 $wgAutoloadClasses['Email\Controller\ForumController'] =  $dir . 'Controller/ForumController.class.php';
 $wgAutoloadClasses['Email\Controller\ReplyForumController'] =  $dir . 'Controller/ForumController.class.php';
 $wgAutoloadClasses['Email\Controller\WeeklyDigestController'] =  $dir . 'Controller/WeeklyDigestController.class.php';
-$wgAutoloadClasses['Email\Controller\AbstractEmailConfirmationController'] =  $dir . 'Controller/EmailConfirmationController.class.php';
+$wgAutoloadClasses['Email\Controller\AbstractEmailConfirmationController'] =  $dir . 'Controller/OtherEmailConfirmationController.class.php';
 $wgAutoloadClasses['Email\Controller\EmailConfirmationController'] =  $dir . 'Controller/EmailConfirmationController.class.php';
-$wgAutoloadClasses['Email\Controller\EmailConfirmationReminderController'] =  $dir . 'Controller/EmailConfirmationController.class.php';
-$wgAutoloadClasses['Email\Controller\ConfirmationChangedEmailController'] = $dir . 'Controller/EmailConfirmationController.class.php';
-$wgAutoloadClasses['Email\Controller\ReactivateAccountController'] =  $dir . 'Controller/EmailConfirmationController.class.php';
+$wgAutoloadClasses['Email\Controller\EmailConfirmationReminderController'] =  $dir . 'Controller/OtherEmailConfirmationController.class.php';
+$wgAutoloadClasses['Email\Controller\ConfirmationChangedEmailController'] = $dir . 'Controller/OtherEmailConfirmationController.class.php';
+$wgAutoloadClasses['Email\Controller\ReactivateAccountController'] =  $dir . 'Controller/OtherEmailConfirmationController.class.php';
 $wgAutoloadClasses['Email\Controller\CategoryAddController'] = $dir . 'Controller/CategoryAddController.class.php';
 $wgAutoloadClasses['Email\Controller\FounderEditController'] =  $dir . 'Controller/FounderController.class.php';
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Controller/FounderController.class.php" 8 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)